package.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "begin-app",
+    "version": "0.0.0",
+    "description": "Begin basic Hello World! app for Node",
+    "arc": {
+      "app": "hello-world",
+      "http": [
+        {
+          "/": {
+            "method": "any",
+            "src": "."
+          }
+        }
+      ]
+    },
+    "scripts": {
+      "start": "cross-env NODE_ENV=testing npx sandbox"
+    },
+    "devDependencies": {
+      "vm2": "3.9.19",
```

Comment:
:exclamation:**Cycode**: Security vulnerability found in newly introduced dependency. 

||| 
 |---|---|
|**Severity**|Critical|
|**Issue**|vm2 Sandbox Escape vulnerability: CVE-2023-37903|
|**Ecosystem**|NPM|
|**Dependency**|`vm2`|
|**Dependency Paths**|`vm2 3.9.19`|
|**Direct Dependency**|Yes|
|**Development Dependency**|Yes|
|**Upgrade**|``|

 
 In vm2 for versions up to 3.9.19, Node.js custom inspect function allows attackers to escape the sandbox and run arbitrary code.

### Impact
Remote Code Execution, assuming the attacker has arbitrary code execution primitive inside the context of vm2 sandbox.

### Patches
None.

### Workarounds
None.

### References
PoC is to be disclosed on or after the 5th of September.

### Similarity with [CVE-2023-37466](https://nvd.nist.gov/vuln/detail/CVE-2023-37466)
While this advisory might look similar to [CVE-2023-37466](https://nvd.nist.gov/vuln/detail/CVE-2023-37466), it is a completely different way of escaping the sandbox.

### For more information
If you have any questions or comments about this advisory:

- Open an issue in [VM2](https://github.com/patriksimek/vm2)

Thanks to [Xion](https://twitter.com/0x10n) (SeungHyun Lee) of [KAIST Hacking Lab](https://kaist-hacking.github.io/) for disclosing this vulnerability.


Would you like to exclude this security vulnerability from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_ignore_package_here** | Applies to this manifest for this package in this request only |
| **#cycode_ignore_package_everywhere** | Applies to this manifest for this package for all requests in your repository |
| **#cycode_ignore_manifest_here** | Applies to this manifest in this request only |

 <!-- e6bfad24-0c71-4f79-a4e1-3fee369ba67c -->